Formula/bitcoin.rb

```diff
@@ -52,6 +52,8 @@ def install
                           "--with-boost-libdir=#{Formula["boost"].opt_lib}",
                           "--prefix=#{prefix}"
     system "make", "install"
+    libexec.install "share/rpcuser"
```

Comment:
Let's do `pkgshare.install "share/rpcuser"` and not symlink it into `bin`. :+1: to suggesting to upstream that it actually be installed into `bin` as `bitcoin-rpcuser` though.